src/EditorFeatures/Core/Implementation/BraceMatching/ClassificationTypeDefinitions.cs

```diff
@@ -6,6 +6,8 @@
 using Microsoft.VisualStudio.Text.Classification;
 using Microsoft.VisualStudio.Utilities;
 
+#pragma warning disable 649
```

Comment:
It would be nice to find a more granular way to do this (maybe an attribute on the specific field), but I don't have any specific suggestions off the top of my head...